lib/internal/cluster/master.js

```diff
@@ -12,6 +12,16 @@ const cluster = new EventEmitter();
 const intercom = new EventEmitter();
 const SCHED_NONE = 1;
 const SCHED_RR = 2;
+var debugPortOffset = 0;
+
+const debugArgRegex = /^--(inspect|debug(?:-brk|port)?)(?:=.*)?$/;
```

Comment:
Nit: Instead of `.*` perhaps it should be `\d+`. That would seem to be consistent with the previous regexp too.